azure-pipelines/build/templates/doclient-lite-native-steps.yml

```diff
@@ -19,6 +19,14 @@ steps:
     arguments: '--project agent --config ${{parameters.config}} --package-for DEB --clean'
   displayName: 'Build agent ${{parameters.targetOsArch}}-${{parameters.config}}'
 
+- task: Bash@3
+  condition: eq('${{parameters.config}}', 'minsizerel')
+  inputs:
+    targetType: 'filePath'
+    filePath: 'build/check_binary_size.sh'
+    arguments: '/tmp/build-deliveryoptimization-agent/linux-${{parameters.config}}/client-lite/deliveryoptimization-agent 2314024'
```

Comment:
>2314024 [](http://example.com/codeflow?start=122&length=7)

mind unpacking this one for me please? what is this

Reply:
Expected size of the binary.

Reply:
so you are clocking it at current, or leaving a bit of room to grow?

Reply:
The script will fail the pipeline if we exceed by 5% or more.